### PR TITLE
createExamenReset

### PR DESCRIPTION
**Pull request by PTsiali**
Fragen backend

### CODE REVIEW

File: Frontend/SPFrontend/src/app/app.module.ts [hunk 1]
```diff
@@ -1,28 +1,57 @@
...
+import {MatOptionModule} from "@angular/material/core";
+import {MatSelect, MatSelectModule} from "@angular/material/select";
+import { CreateExamComponent } from './components/create-exam/create-exam.component';
+import {MatExpansionModule} from '@angular/material/expansion';
+import {MatGridListModule} from "@angular/material/grid-list";
 
 @NgModule({
   declarations: [
     AppComponent,
-    HomePageComponent,
```
Comment (Nevermint) [inline Frontend/SPFrontend/src/app/app.module.ts]: Warum wurde HomePageComponent entfernt?

File: Frontend/SPFrontend/src/app/app.component.html [hunk 2]
```diff
@@ -1 +1,3 @@
 <router-outlet></router-outlet>
+<app-create-exam></app-create-exam>
```
Comment (Nevermint) [inline Frontend/SPFrontend/src/app/app.component.html:1]: Der Verweis zum NAvigieren muss ins routing.modules.ts

**Review by ghost — COMMENTED** [review]
Rename Pojo to DTO & Rebase with main

File: Backend/src/main/java/de/hft_stuttgart/winf/proj2/sp/backend/handler/QuestionsHandler.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        try {
+            DbQuestions dbAccess = new DbQuestions();
+            return dbAccess.getQuestions(module);
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return null;
+    }
+
+    @Path("newQuestion")
```
Comment (MediocreMarcel) [inline Backend/src/main/java/de/hft_stuttgart/winf/proj2/sp/backend/handler/QuestionsHandler.java:99]: Please add javadoc

File: Backend/src/main/java/de/hft_stuttgart/winf/proj2/sp/backend/handler/QuestionsHandler.java [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+            DbModule dbAccess = new DbModule();
+            if (!dbAccess.createCourses(module, userId)){
+                return Response.status(Response.Status.CONFLICT).build();
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return Response.ok().build();
+    }
+
```
Comment (MediocreMarcel) [inline Backend/src/main/java/de/hft_stuttgart/winf/proj2/sp/backend/handler/QuestionsHandler.java:72]: please add javadoc

**Review by MediocreMarcel — CHANGES_REQUESTED** [review]

File: Backend/src/main/java/de/hft_stuttgart/winf/proj2/sp/backend/Pojo/QuestionsPojo.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
+package de.hft_stuttgart.winf.proj2.sp.backend.Pojo;
+import de.hft_stuttgart.winf.proj2.sp.backend.util.Column;
+import lombok.*;
+
+
+@AllArgsConstructor
+@Getter
+@ToString
+@EqualsAndHashCode
+public class QuestionsPojo {
```
Comment (MediocreMarcel) [inline Backend/src/main/java/de/hft_stuttgart/winf/proj2/sp/backend/Pojo/QuestionsPojo.java]: Rename to DTO also move to dto folder

File: Backend/src/main/java/de/hft_stuttgart/winf/proj2/sp/backend/dao/ModuleDao.java [hunk 6]
```diff
@@ -0,0 +1,23 @@
+package de.hft_stuttgart.winf.proj2.sp.backend.dao;
+
+import de.hft_stuttgart.winf.proj2.sp.backend.util.Column;
+import lombok.*;
+
+@AllArgsConstructor
+@Getter
+@ToString
+@EqualsAndHashCode
+public class ModuleDao {
```
Comment (MediocreMarcel) [inline Backend/src/main/java/de/hft_stuttgart/winf/proj2/sp/backend/dao/ModuleDao.java]: Change to DTO, also change folder

File: Backend/src/main/java/de/hft_stuttgart/winf/proj2/sp/backend/dao/UserDao.java [hunk 7]
```diff
@@ -0,0 +1,21 @@
+package de.hft_stuttgart.winf.proj2.sp.backend.dao;
+
+import lombok.*;
+
+@AllArgsConstructor
+@Getter
+@ToString
+@EqualsAndHashCode
+public class UserDao {
```
Comment (MediocreMarcel) [inline Backend/src/main/java/de/hft_stuttgart/winf/proj2/sp/backend/dao/UserDao.java]: Change to dto and move to folder

**Review by MediocreMarcel — CHANGES_REQUESTED** [review]

**Pull request closed without merging**